include/class.format.php

```diff
@@ -228,7 +228,7 @@ function safe_html($html) {
     function localizeInlineImages($text) {
         // Change file.php urls back to content-id's
         return preg_replace(
-            '/src="(?:\/[^"]+?)?\/file\\.php\\?(?:\w+=[^&]+&(?:amp;)?)*?key=([^&]+)[^"]*/',
+            '`src="(?:https?:/)?(?:/[^/"]+)*?/file\\.php\\?(?:\w+=[^&]+&(?:amp;)?)*?key=([^&]+)[^"]*`',
```

Comment:
I fixed this by using a workaround
I just clicked the HTML button and used the image source. I have the image on my word press site and just used <img src="image url goes here"> worked for me.